include/h2o/string_.h

```diff
@@ -178,6 +178,11 @@ inline int h2o_lcstris(const char *target, size_t target_len, const char *test,
     return h2o__lcstris_core(target, test, test_len);
 }
 
+inline size_t h2o_base64_encode_capacity(unsigned len)
```

Comment:
The reason you are seeing CI failure is because the function is not defined is `static`. Since it is not defined as `static`, the function is defined multiple times as each source file includes the header file, ending up with a multiple definition error.

The rule I'd like you to follow is to _declare_ a function as `static` first, and then _define_ a function as `inline`. For example, please see the occurrences of `h2o_lcstris` (you'd find the declaration and the definition within the header file).